dll/win32/shell32/CDefView.cpp

```diff
@@ -757,20 +793,21 @@ BOOL CDefView::CreateList()
     TRACE("%p\n", this);
 
     dwStyle = WS_TABSTOP | WS_VISIBLE | WS_CHILDWINDOW | WS_CLIPSIBLINGS | WS_CLIPCHILDREN |
-              LVS_SHAREIMAGELISTS | LVS_EDITLABELS | LVS_AUTOARRANGE; // FIXME: Why is LVS_AUTOARRANGE here?
-    dwExStyle = WS_EX_CLIENTEDGE;
-    ListExStyle = 0;
+              LVS_SHAREIMAGELISTS | LVS_EDITLABELS | LVS_AUTOARRANGE; // FIXME: Remove LVS_AUTOARRANGE when the view is able to save ItemPos
+    dwExStyle = (m_FolderSettings.fFlags & FWF_NOCLIENTEDGE) ? 0 : WS_EX_CLIENTEDGE;
+    ListExStyle = LVS_EX_INFOTIP | LVS_EX_LABELTIP;
 
     if (m_FolderSettings.fFlags & FWF_DESKTOP)
     {
         m_FolderSettings.fFlags |= FWF_NOCLIENTEDGE | FWF_NOSCROLL;
         dwStyle |= LVS_ALIGNLEFT;
+        // LVS_EX_REGIONAL?
```

Comment:
It seems this style is somewhat deprecated (and would work only on LVS_ICON styled listviews).
Does Windows XP/2003 set it?

Reply:
I think it's used on the desktop listview, perhaps only on multimon...